[TEST-FAILURE] QueueStatisticsTest.testRejectedOfferOperationCount

com.hazelcast.queue.QueueStatisticsTest.testRejectedOfferOperationCount
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.maintenance-OpenJDK6/com.hazelcast$hazelcast/14/testReport/junit/com.hazelcast.queue/QueueStatisticsTest/testRejectedOfferOperationCount/

```
java.lang.AssertionError: expected:<10> but was:<9>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:743)
    at org.junit.Assert.assertEquals(Assert.java:118)
    at org.junit.Assert.assertEquals(Assert.java:555)
    at org.junit.Assert.assertEquals(Assert.java:542)
    at com.hazelcast.queue.QueueStatisticsTest.testRejectedOfferOperationCount(QueueStatisticsTest.java:64)
```

--- a/hazelcast/src/test/java/com/hazelcast/queue/QueueStatisticsTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/queue/QueueStatisticsTest.java
@@ -45,8 +45,14 @@ public class QueueStatisticsTest extends AbstractQueueTest {
             queue.put("item" + i);
         }
 
-        LocalQueueStats stats = queue.getLocalQueueStats();
-        assertEquals(30, stats.getOfferOperationCount());
+        final LocalQueueStats stats = queue.getLocalQueueStats();
+        AssertTask task = new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertEquals(30, stats.getOfferOperationCount());
+            }
+        };
+        assertTrueEventually(task);
     }
 
     @Test
@@ -60,8 +66,15 @@ public class QueueStatisticsTest extends AbstractQueueTest {
             queue.offer("item" + i);
         }
 
-        LocalQueueStats stats = queue.getLocalQueueStats();
-        assertEquals(10, stats.getRejectedOfferOperationCount());
+        final LocalQueueStats stats = queue.getLocalQueueStats();
+        AssertTask task = new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertEquals(10, stats.getRejectedOfferOperationCount());
+            }
+        };
+        assertTrueEventually(task);
+
     }
 
 

